src/components/button-row/button-row.js

```diff
@@ -47,9 +32,33 @@ class ButtonRow extends React.Component {
     return components;
   }
 
+  /**
+  Creates the inline list based on the provided props.
+  @method _generateButtonsList
```

Comment:
`@method` is not required

Reply:
Why not? This is a method, is it not?

Reply:
jsdoc doesn't require that tag any longer.